### ISSUE TITLE
Move flow temperature back to settings page

### ISSUE THREAD

**Issue by janhjordie** [OlivierZal/com.melcloud#30]
Hey @OlivierZal 
A thing I liked with the old version was having the heat flow target temperature and water tank target temperature on the settings page.
Normally you won't change the water tank temperature and water flow temperature as a daily routine. It's normally a set and forget.
That's also why it's keept on the Unit Settings in the real MELCloud app.

Maybe it's a bit difficult since there can be several zones.

Just a thought.

![image](https://user-images.githubusercontent.com/79910/197585950-e9498d22-4e81-4b29-beee-729d3b0f0161.png)
![image](https://user-images.githubusercontent.com/79910/197586194-08fb22ef-11e2-467d-91e7-899a8ac80a12.png)

**Comment by OlivierZal** [comment]
Hi @janhjordie, thanks for your feedback. As usual I always appreciate it. 

I removed flow temperatures from settings for technical and design reasons.

A first kind of reasons is related to Homey: capabilities can be "listened", not settings. To bypass this, a hybrid system was created with settings duplicated by capabilities: design was difficult to read and then difficult to maintain, in particular at the flow card level.  A lot of "sleeps" were necessary to make sure settings and capabilities were synchronized. 

Also using Homey standard capabilities is easier since some flow cards are automatically created this way. 

The second reason is related to MELCloud API. Previously 2 Homey devices needed to be created in case of 2-zone heat pumps, but since then I've learnt how MELCloud works:
- some values are shared between zones
- some other values are specific to only one zone but needs to be changed together with the same for the other zone, otherwise the other is set to 0 (what is absurd, but it's because those values share the same "effective flag")

To avoid this, all the values needed to be gathered under a single Homey device. 
Another advantage of capabilities over settings is that settings are fixed, capabilities can be added/removed. 

Then, after having carefully read the Homey documentation, I succeeded to manage 1-zone and 2-zone heat pumps, but also those handling cool flows, with only 1 device, seamless for end users. 

For example: today you see only "heat flow settings" for zone 1, and the day you decide to activate zone 2, you'll see new categories in your Homey without having to pair your device again. And if someone have a heat pump handling cool flows, his drop-down will display all the "cool flow settings" you don't see on your Homey. 

I hope this explains why I changed the design.
